web/content/js/nav.js

```diff
@@ -67,15 +58,57 @@ let navItems = [
 		href: "/Application_Centric_Deployment/index-foreman-el.html"
 	}]
 },{
-	title: "Katello documentation", items: [
+	title: "Foreman on Debian", items: [
+	{
+		title: "Release Notes",
+		href: "/Release_notes/index-foreman-deb.html"
+	},{
+		title: "Quickstart Guide",
+		href: "/Quickstart_Guide/index-foreman-deb.html"
+	},{
+		title: "Installation Guide",
+		href: "/Installing_Server_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Installing Smart Proxy",
+		href: "/Installing_Proxy_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Upgrading and Updating",
+		href: "/Upgrading_and_Updating/index-foreman-deb.html"
+	},{
+		title: "Deploying Foreman on AWS",
+		href: "/Deploying_on_AWS/index-foreman-deb.html"
+	},{
+		title: "Configuring Smart Proxies with a Load Balancer",
+		href: "/Configuring_Load_Balancer/index-foreman-deb.html"
```

Comment:
Katello specific guide in all the installer commands and certificate handling. I'd recommend to not link this (nor in the foreman-el flavor).

Reply:
That's what I thought.

Reply:
Can I actually move it to Katello? I don't want to completely remove a guide from all menus, in Katello this might be pretty accurate I think.